presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -151,8 +152,8 @@ private static ClassLoader findPlatformClassLoader()
             return (ClassLoader) method.invoke(null);
         }
         catch (NoSuchMethodException ignored) {
-            // use null class loader on Java 8
-            return null;
+            // use extension class loader which is the parent of Launcher.class classloader on Java < 9
```

Comment:
Maybe:
```
use parent of launcher classloader which is ExtClassLoader so java extensions class are available (lib-ext.jar)
```
?

Reply:
What is `lib-ext.jar`? That doesn't seem to exist

Reply:
We should mention that this is needed because the JCE is part of the extension class loader in JDK 8.